workshop/exercise-7/README.md

```diff
@@ -95,42 +94,41 @@ A `PipelineResource` is used to define artifacts that are passed into and out of
           command:
             - /kaniko/executor
           args:
-            - --dockerfile=${inputs.params.pathToDockerFile}
-            - --destination=${outputs.resources.builtImage.url}
-            - --context=${inputs.params.pathToContext}
+            - --dockerfile=$(params.pathToDockerFile)
+            - --destination=$(outputs.resources.builtImage.url)
+            - --context=$(params.pathToContext)
     ```
 
 2. Apply this yaml file to your kubernetes cluster.
     ```
     kubectl apply -f image-from-source-task.yaml
     ```
   
-3. Now that our `Task` is defined, let's define a `TaskRun` that will run the `Task`. This `TaskRun` will bind inputs and outputs to our earlier defined resources, `fib-knative` and `fib-knative-git`. This `TaskRun` will also execute the `steps` we defined for our `Task`. Take a look at this file, called `image-from-source-task-run.yaml`.
+3. Now that our `Task` is defined, let's define a `TaskRun` that will run the `Task`. This `TaskRun` will bind inputs and outputs to our earlier defined resources, `fib-knative` and `fib-knative-git`. This `TaskRun` will also execute the `steps` we defined for our `Task`. We also define a `serviceAccount` for this `TaskRun` to utilize for any required permissions. Take a look at this file, called `image-from-source-task-run.yaml`.
```

Comment:
I wonder if we need to say a little bit more about ServiceAccounts?  I don't want to dive too deep, but this one sentence might be a bit too little.

Reply:
we give a little explanation the first time service accounts are mentioned, in exercise 6:
```
5. A `Service Account` provides an identity for processes that run in a Pod. This Service Account will be used to link the build process for Knative to the Secrets you just created. View the service account file, and notice that it's using the credentials you created earlier for pulling from and pushing to the container registry:
```